sklearn/model_selection/_search.py

```diff
@@ -908,7 +908,9 @@ def fit(self, X, y=None, **params):
             If a fit parameter is an array-like whose length is equal to
             `num_samples` then it will be split across CV groups along with `X`
             and `y`. For example, the :term:`sample_weight` parameter is split
```

Comment:
I just realized that the use of "groups" in this paragraph is really misleading.

```suggestion
            If a fit parameter is an array-like whose length is equal to
            `num_samples` then it will be split by cross-validation along with `X`
            and `y`. For example, the :term:`sample_weight` parameter is split
```